packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -47,6 +47,27 @@ export interface IScrollablePaneProps extends React.HTMLAttributes<HTMLElement |
   initialScrollPosition?: number;
 
   scrollbarVisibility?: ScrollbarVisibility;
+
+  /**
+   * If true, it optimizes the performance, but may affect component behavior.
+   * It is suggested to test the component so that it works as per the desired behavior.
+   * Set it to true only if ScrollablePane has Sticky component(s) &
+   * for no Sticky component, prop 'stickyPosition' is 'StickyPositionType.Both'.
+   *
+   * It assumes:
+   * 1. 'OnScroll' sticky behavior for all Sticky components, props
```

Comment:
modify comment to- 'OnScroll' sticky behavior for all Sticky components, having 'stickyPosition' prop equal to StickyPosition.Header